lib/reporters/mini.js

```diff
@@ -216,7 +216,16 @@ MiniReporter.prototype.finish = function (runStatus) {
 	}
 
 	if (runStatus.failFastEnabled === true) {
-		status += '\n\n  ' + colors.failFast('`--fail-fast` is on. Any number of tests may have been skipped');
+		status += '\n\n  ' + colors.information('`--fail-fast` is on. Any number of tests may have been skipped');
+	}
+
+	if (runStatus.hasExclusive === true) {
+		var remainingTests = runStatus.testCount - (runStatus.passCount + runStatus.failCount);
+		if (remainingTests > 0) {
+			status += '\n\n  ' + colors.information('The .only() modifier is used in some tests. At least', remainingTests, 'tests were not run.');
```

Comment:
Meh, it still needs to be wrapped with `colors.information()`, so this combination seems fine to me.

Previously we were targeting Node.js 0.10, which meant that we couldn't use template strings. Now we can, but it's no big deal if new code follows the pattern already used in the file.